src/utils/date.js

```diff
@@ -62,7 +62,13 @@ export function getDateFromFirstdayParam(firstDayParam) {
 		return dateFactory()
 	}
 
-	return new Date(firstDayParam)
+	const [year, month, date] = firstDayParam.split('-')
+		.map((str) => parseInt(str, 10))
```

Comment:
No handling of bad value for `firstDayParam` (not number for instance) ?

Reply:
How should we handle that case?
Just default to today and log an error message?

Reply:
Seems fine to me. :+1: